Parser.py

```diff
@@ -119,7 +106,7 @@ def signs_missing_from_of(modified_lines):
     for variables in range(len(of_range)):
         if variables == 0:
             matched = re.match("^([-+]?)((\d+(\.\d+)?)?)(\*?)x(\d+)", temp_group)
-        if variables != 0:
+        else:

```

Comment:
Also in line 139 of changed (152 in original)

Reply:
![image](https://user-images.githubusercontent.com/12659251/79690182-c6fb0680-8261-11ea-9b56-0ca75004b2b9.png)

PyCharm detects code duplication. I thought the `else` would fix it, but if I change it in line 152 as well, it warns us again. The two functions have to be refactored somehow... 😕